src/app/components/shell/footer/footer.vue

```diff
@@ -10,9 +10,7 @@
 						<br class="visible-xs" />
 						Test the beta version of the Game Jolt app.
 					</p>
-					<p>
-						<app-app-buttons source="footer" />
-					</p>
+					<app-app-buttons source="footer" centered />
```

Comment:
Is `centered` meant to be a prop, or are we assigning styling to it through this somewhere? Is this meant to be using the `justified` prop instead?

Reply:
This also seems to be requiring a `source!: AppPromotionSource` prop.

Reply:
good catch, the centered was old. also, source is passed